SSA/Projects/InstCombine/Test.lean

```diff
@@ -87,19 +88,14 @@ def ops' := [op0, op1, op2, op3, op4]
 #eval mkExpr    (Γn 5)  (ops.get! 4) ["3", "2", "1", "0", "arg0"]
 #eval mkReturn  (Γn 6)  (ops.get! 5) ["4", "3", "2", "1", "0", "arg0"]
 
-#eval (mkCom bb0).toOption
+def com := mkCom (d := InstCombine.MetaLLVM 0) bb0 |>.toOption |>.get (by rfl)
 
-
-#check ExceptT.run
-
-def com := mkCom bb0 |>.toOption |>.get (by rfl)
-
-#eval com
+#reduce com
```

Comment:
I think we should either drop the `#reduce` here altogether, or put it behind a `#guard_msgs` line to make sure whatever the `#reduce` is checking is automatically tested (and not polluting the output of `lake build`)

Reply:
This has been fixed at: https://github.com/opencompl/ssa/pull/337. I had planned to resolve this in this PR, but we merged it before doing so.